apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -86,7 +86,7 @@ public sealed class CloudTrace : IDisposable
         /// This <see cref="IManagedTracer"/> is a singleton.
         /// </summary>
         public static readonly IManagedTracer Tracer = ManagedTracer.CreateDelegatingTracer(
-            () => ContextInstanceManager.Get<IManagedTracer>() ?? NullManagedTracer.Instance);
+            () => ContextTracerManager.GetCurrentTracer());
```

Comment:
Unwrap onto the previous line now that it's much shorter?

Reply:
Done